install_config/install/advanced_install.adoc

```diff
@@ -2170,102 +2170,71 @@ ifdef::openshift-origin[]
 *openshift/origin-ansible*
 endif::[]
 image is a containerized version of the {product-title} installer that runs as a
-link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/managing_containers/running_system_containers[system container]. System containers are stored and run outside of the traditional
-*docker* service. Functionally, using the containerized installer is the same as
-using the traditional RPM-based installer, except it is running in a
-containerized environment instead of directly on the host.
-
-. Use the Docker CLI to pull the image locally:
-+
-----
-ifdef::openshift-enterprise[]
-$ docker pull registry.access.redhat.com/openshift3/ose-ansible:v3.6
-endif::[]
-ifdef::openshift-origin[]
-$ docker pull docker.io/openshift/origin-ansible:v3.6
-endif::[]
-----
-
-. The installer system container must be stored in
-link:https://access.redhat.com/documentation/en-us/red_hat_satellite/6.2/html/content_management_guide/managing_ostree_content[OSTree]
-instead of defaulting to *docker* daemon storage. Use the Atomic CLI to import
-the installer image from the local *docker* engine to OSTree storage:
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/managing_containers/running_system_containers[system container].
+System containers are stored and run outside of the traditional
+*docker* service. The installer image provides the same functionality
+as the traditional RPM-based installer, but it runs in a containerized
+environment that provides all of its dependencies rather than being
+installed directly on the host.
+
+. As the root user, use the Atomic CLI to run the system container as a one-time systemd service:
 +
 ----
-$ atomic pull --storage ostree \
-ifdef::openshift-enterprise[]
-    docker:registry.access.redhat.com/openshift3/ose-ansible:v3.6
-endif::[]
-ifdef::openshift-origin[]
-    docker:docker.io/openshift/origin-ansible:v3.6
-endif::[]
-----
-
-. Install the system container so it is set up as a systemd service:
-+
-----
-$ atomic install --system \
+# atomic install --system \
     --storage=ostree \
     --name=openshift-installer \//<1>
     --set INVENTORY_FILE=/path/to/inventory \//<2>
 ifdef::openshift-enterprise[]
-    docker:registry.access.redhat.com/openshift3/ose-ansible:v3.6
+    registry.access.redhat.com/openshift3/ose-ansible:v3.6
```

Comment:
@ashcrow @giuseppe check me on this -- it looks like the latest version of atomic on RHEL now has more of the functionality you wanted here.

Previously, it needed to pull through docker, and it would import into ostree every time, and created a regular systemd service (not one-time, and no output, just journald, and service definition was retained after running).

That is still the behavior AFAICS if I specify the image with the `docker:` prefix, however if I leave that off, then it directly imports the image to ostree (no docker even running), creates a one-time service, runs it, and uninstalls it, which I think is the behavior you were wanting to get to.

Just wanted you to check if I'm missing anything here.

Reply:
> [...]if I leave that off, then it directly imports the image to ostree (no docker even running), creates a one-time service, runs it, and uninstalls it, which I think is the behavior you were wanting to get to.

That is the ``atomic.run`` ``once`` feature that is much cleaner than the original one. Looks like RHEL 7.4 has atomic 1.18.1-3 which has the run once ability. I'm not sure if the docker import is still needed or not with that version. @giuseppe?

Reply:
It looks like with that feature, it doesn't even use the service name provided. I don't think it ever creates a systemd service per se, doesn't seem to send anything to journald. So I'll take out all the discussion of systemd service, which should make this even simpler :)